pandas/core/indexes/multi.py

```diff
@@ -1541,8 +1540,7 @@ def _get_level_values(self, level, unique=False):
 
     def get_level_values(self, level):
         """
-        Return vector of label values for requested level,
```

Comment:
I think should keep the extra information here and in the other docstrings rather than just removing.